addons/sitting/functions/fnc_sit.sqf

```diff
@@ -1,22 +1,24 @@
 #include "script_component.hpp"
 /*
- * Author: Jonpas
+ * Author: Jonpas, vabene1111
  * Sits down the player.
  *
  * Arguments:
  * 0: Seat <OBJECT>
  * 1: Player <OBJECT>
+ * 2: Seat Position <NUMBER> (default: 0)
  *
  * Return Value:
  * None
  *
  * Example:
- * [seat, player] call acex_sitting_fnc_sit;
+ * [seat, player] call acex_sitting_fnc_sit
```

Comment:
`cursorObject` instead of `seat` in all functions examples would be much more useful

Reply:
It's an object, that's all that matters, you can get it in thousands of different ways. :)

Reply:
I meant for copy-paste, just paste to debug console and hit `EXEC`

Reply:
having code example you can use instantly is always nice

Reply:
Good point, I am always too lazy with copy/pasting. :laughing: